pages/dkp/kommander/2.2/dkp-upgrade/upgrade-kommander/index.md

```diff
@@ -39,6 +39,30 @@ This section describes how to upgrade your Kommander Management cluster and all
   ```bash
   wget "https://downloads.d2iq.com/dkp/v2.2.0/dkp-catalog-applications-charts-bundle-v2.2.0.tar.gz"
   ```
+
+-   For clusters upgrading from 2.1.1 with HTTP Proxy installed:
+
+    Edit the `gatekeeper-overrides`, and add a new configuration property `disableMutation` with the value `false`. The Gatekeeper configuration has changed between versions `v2.1.1` and `v2.2.0`:
+
+    ```yaml
+      disableMutation: false # <-- this value is new in 2.2.0 and is missing in documentation
```

Comment:
I'm not sure I understand the comment here, aren't we adding this to the documentation in this PR?  :-) 

```suggestion
      disableMutation: false # <-- this value is new in 2.2.0
```

Reply:
Correct, this was copied from my notes. Good catch! :smile: